ajax/libs/cleanslate/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "cleanslate",
+  "filename": "cleanslate.min.css",
+  "version": "0.10.1",
+  "homepage": "http://cleanslatecss.com",
+  "description": "Extreme CSS reset stylesheet",
+  "keywords": [
+    "reset css"
```

Comment:
Why not use the keywords from upstream? https://github.com/premasagar/cleanslate/blob/master/component.json#L6